mackup/applications/thunderbird.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = Thunderbird
+
+[configuration_files]
+.thunderbird
```

Comment:
Good point. I guess we should not sync the "Mail" folders. However, how to do that given that the profile folder contains a random name?

![image](https://cloud.githubusercontent.com/assets/170947/22139454/49901ca2-dee6-11e6-82c8-eeb1a34d23f6.png)